libs/model/src/aggregates/token/utils.ts

```diff
@@ -53,6 +94,17 @@ export async function handleCapReached(
         lpBondingCurveAddress,
         privateKey: config.WEB3.LAUNCHPAD_PRIVATE_KEY!,
       });
+
+      if (notifyUsers.length > 0) {
+        await provider.triggerWorkflow({
+          key: WorkflowKeys.LaunchpadCapReached,
+          users: notifyUsers,
+          data: {
+            symbol: tokenHolders[0].symbol,
```

Comment:
There is no community id in the data but in `packages/commonwealth/client/scripts/views/components/KnockNotifications/CustomNotificationCell.tsx` we expect there to be so that we can construct a link.